com.github.inercia.k3x.json

```diff
@@ -0,0 +1,120 @@
+{
+  "app-id": "com.github.inercia.k3x",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "k3x",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=ssh-auth",
+    "--share=network",
+    "--env=LD_LIBRARY_PATH=/app/lib:/app/lib/x86_64-linux-gnu:/lib64:/lib:/usr/lib64:/usr/lib:/usr/local/lib",
+    "--filesystem=/run/docker.sock",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=com.canonical.indicator.application",
+    "--talk-name=org.ayatana.indicator.application",
+    "--filesystem=home:rw"
+  ],
+  "cleanup": [
+    "/bin/intltool-*",
+    "/etc/bash_completion.d",
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/aclocal",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    "shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json",
+    {
+      "name": "libappindicator"
+    },
+    {
+      "name": "granite",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/elementary/granite.git",
+          "tag": "5.3.1"
+        }
+      ]
+    },
+    {
+      "name": "elementary-icons",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/elementary/icons.git",
+          "tag": "5.0"
+        }
+      ]
+    },
+    {
+      "name": "kubectl",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mv -f kubectl /app/kubectl",
+        "chmod 755 /app/kubectl"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl",
+          "sha256": "bb16739fcad964c197752200ff89d89aad7b118cb1de5725dc53fe924c40e3f7"
+        }
+      ]
+    },
+    {
+      "name": "requirements",
+      "buildsystem": "simple",
+      "build-options": {
+        "build-args": [
+          "--share=network"
```

Comment:
Flathub builds are offline to ensure reproducibility; you can generate JSON snippet for these libraries with https://github.com/flatpak/flatpak-builder-tools/tree/master/pip.